include/enums.h

```diff
@@ -549,119 +549,119 @@ enum SoundIDs {
-    SOUND_009B                                  = 0x0000009B,
-    SOUND_009D                                  = 0x0000009D,
-    SOUND_LRAW_009E                             = 0x0000009E,
-    SOUND_LRAW_009F                             = 0x0000009F,
-    SOUND_LRAW_00A0                             = 0x000000A0,
-    SOUND_00A1                                  = 0x000000A1,
-    SOUND_LRAW_00A2                             = 0x000000A2,
-    SOUND_00A3                                  = 0x000000A3,
-    SOUND_00A4                                  = 0x000000A4,
-    SOUND_00A5                                  = 0x000000A5,
-    SOUND_00A6                                  = 0x000000A6,
-    SOUND_00A7                                  = 0x000000A7,
-    SOUND_00A8                                  = 0x000000A8,
-    SOUND_00A9                                  = 0x000000A9,
-    SOUND_00AA                                  = 0x000000AA,
-    SOUND_00AB                                  = 0x000000AB,
-    SOUND_00AC                                  = 0x000000AC,
-    SOUND_LRAW_00AD                             = 0x000000AD,
-    SOUND_LRAW_00AE                             = 0x000000AE,
-    SOUND_LRAW_00AF                             = 0x000000AF,
+    SOUND_OBJECT_LAND                           = 0x00000048,
+    SOUND_LRAW_004A                               = 0x0000004A,
+    SOUND_LRAW_004C                               = 0x0000004C,
+    SOUND_LRAW_004D                               = 0x0000004D,
+    SOUND_LRAW_004E                               = 0x0000004E,
+    SOUND_LRAW_004F                               = 0x0000004F,
+    SOUND_LRAW_0050                               = 0x00000050,
+    SOUND_0051                                    = 0x00000051,
+    SOUND_LRAW_0052                               = 0x00000052,
+    SOUND_0053                                    = 0x00000053,
+    SOUND_LRAW_0054                               = 0x00000054,
+    SOUND_0055                                    = 0x00000055,
+    SOUND_LRAW_0056                               = 0x00000056,
+    SOUND_0057                                    = 0x00000057,
+    SOUND_0058                                    = 0x00000058,
+    SOUND_0059                                    = 0x00000059,
+    SOUND_005A                                    = 0x0000005A,
+    SOUND_LRAW_005B                               = 0x0000005B,
+    SOUND_005C                                    = 0x0000005C,
+    SOUND_LRAW_005D                               = 0x0000005D,
+    SOUND_005E                                    = 0x0000005E,
+    SOUND_005F                                    = 0x0000005F,
+    SOUND_LRAW_0060                               = 0x00000060,
+    SOUND_0061                                    = 0x00000061,
+    SOUND_0062                                    = 0x00000062,
+    SOUND_0064                                    = 0x00000064,
```

Comment:
Too much whitespace here?

Reply:
I intentionally added two extra spaces for sounds yet-to-be-named so I can spot them more quickly.